tests/test_offchain.py

```diff
@@ -29,9 +29,11 @@ def test_prepare_transaction_raises():
 def test_prepare_create_transaction_default(alice_pubkey):
     from bigchaindb_driver.offchain import prepare_create_transaction
     create_transaction = prepare_create_transaction(signers=alice_pubkey)
+    del create_transaction['asset']['id']
```

Comment:
I've had to add these to make tests pass until bigchaindb/bigchaindb#1101 is fixed.